src/organizations/components/OrgProfileTab/LeaveOrg.tsx

```diff
@@ -1,36 +1,41 @@
 // Libraries
 import React, {FC, useContext} from 'react'
-import {useSelector} from 'react-redux'
+import {useDispatch, useSelector} from 'react-redux'
 
 // Components
-import {
-  ButtonShape,
-  ComponentColor,
-  ConfirmationButton,
-  FlexBox,
-  IconFont,
-} from '@influxdata/clockface'
+import {Button, ComponentColor, FlexBox, IconFont} from '@influxdata/clockface'
 
 // Selector
 import {selectCurrentOrg, selectUser} from 'src/identity/selectors'
 
 // Providers
 import {UsersContext} from 'src/users/context/users'
 
-// Constants
-import {CLOUD_URL} from 'src/shared/constants'
+// Overlay
+import {dismissOverlay, showOverlay} from 'src/overlays/actions/overlays'
+
+// Types
+import {RemoveMemberOverlayParams} from 'src/users/components/RemoveMemberOverlay'
 
 // Styles
 import 'src/organizations/components/OrgProfileTab/style.scss'
 
 export const LeaveOrgButton: FC = () => {
   const org = useSelector(selectCurrentOrg)
+  const {users} = useContext(UsersContext)
   const currentUserId = useSelector(selectUser)?.id
   const {removeUser} = useContext(UsersContext)
+  const userToRemove = users.find(user => user.id === currentUserId)
+  const dispatch = useDispatch()
 
   const handleRemoveUser = () => {
-    removeUser(currentUserId)
-    window.location.href = CLOUD_URL
+    const params: RemoveMemberOverlayParams = {
+      removeUser,
+      users,
+      userToRemove,
+    }
```

Comment:
The function for removing users lives in the User Context, and the overlay (which lives in a distinct DOM node hierarchy rendered by a React portal) doesn't have access to the provider. IMHO, the best and cleanest way to let us access that function through our overlay setup is simply to pass the function (and any data it needs) in the params object that our overlay controller receives. Here are the alternatives, and why they weren't picked.

1. Make the context functions into redux actions, and move the associated logic into a reducer. 

This would be more coherent and flexible than using a context, but requires a huge restructuring of the app.

2. Wrap the user provider around our overlay framework.

Bad. We don't need a separate copy of the users context in the app. Wasteful, and may cause unexpected re-renders. Also, this likely won't work since the contexts will be separate, so modifications to one won't impact the other until the page is reloaded.